pkg/arvo/app/btc-wallet.hoon

```diff
@@ -601,7 +601,7 @@
           %=  state
               pend.piym     (~(del by pend.piym) txid)
               ps.piym       (~(del by ps.piym) payer)
-              num-fam.piym  (~(put by num-fam.piym) payer (dec nf))
+              num-fam.piym  (~(put by num-fam.piym) payer (?:((gth nf 0) (dec nf) 0)))
```

Comment:
The ubuntu build (which runs the full arvo test suite) is failing because this file is failing to build, because there are extra parens around the `?:`.

Also, I'd skip the function call and just compare to `0` directly: `?:(=(0 nf) 0 (dec n))`.

Reply:
Thanks Joe, took your suggestion and pushed the change up.